next/backend/forms/test/uiSchema.json

```diff
@@ -158,7 +158,8 @@
       "ui:widget": "RadioButton",
       "ui:label": false,
       "ui:options": {
-        "className": "flex flex-col gap-4",
+        "className": "flex flex-row gap-4 mt-4",
+        "//className": "flex flex-col gap-4 mt-4 max-w-[500px]",
```

Comment:
is this on purpose or an error ? I didn't find this kind of syntax elsewhere

Reply:
if you mean line 162 `"//className": "flex flex-col gap-4 mt-4 max-w-[500px]",` it is not an error, its just comment in json file how to use styling  if we want to use radio button in column order instead of row

Reply:
Ok, get it, wasn't familiar with commenting this way. Lgtm.